WhaleSpotting/react-app/src/components/Profile.tsx

```diff
@@ -105,7 +132,7 @@ export function Profile(): JSX.Element {
             <div className="feed">
                 <h2 className="heading">Your {feedToggle}</h2>
                 <div className="card-holder">
-                    {cards.length === 0 && page === 1 && feedToggle === "Sightings" ? <div className="card-component">Nothing here, <Link to="reportsighting"> report a sighting </ Link> </div> : cards}
+                    {cards.length === 0 && page === 1 && feedToggle === "" ? <div className="card-component">Nothing here, <Link to="reportsighting"> report a sighting </ Link> </div> : cards}
```

Comment:
So this wont show anything for Sightings?

Reply:
Only when there are no sightings submitted by the user, it will show the link to direct them to Report sighting (on the sighting feed), so I think we need feedToggle === "Sightings". And I think it works before.

Reply:
![image](https://user-images.githubusercontent.com/67764574/129219653-e341a136-49f0-46e9-92a0-be078770e2bf.png)

Reply:
with feedToggle === "", it look like this.
![image](https://user-images.githubusercontent.com/67764574/129219928-be9627f3-84c5-4ad6-83f2-3726761b6a17.png)